tests/native_api_tests/test_api_model_update.cpp

```diff
@@ -293,188 +297,103 @@ constexpr double i_shunt = 0.015 / 0.025 * i;
-
-        result.add_buffer("link", std::ssize(link_id), std::ssize(link_id), nullptr, nullptr);
-        result.add_attribute_buffer("link", "id", link_id.data());
-        result.add_attribute_buffer("link", "from_node", link_from_node.data());
-        result.add_attribute_buffer("link", "to_node", link_to_node.data());
-        result.add_attribute_buffer("link", "from_status", link_from_status.data());
-        result.add_attribute_buffer("link", "to_status", link_to_status.data());
-
-        result.add_buffer("source", std::ssize(source_id), std::ssize(source_id), nullptr, nullptr);
-        result.add_attribute_buffer("source", "id", source_id.data());
-        result.add_attribute_buffer("source", "node", source_node.data());
-        result.add_attribute_buffer("source", "status", source_status.data());
-        result.add_attribute_buffer("source", "u_ref", source_u_ref.data());
-        result.add_attribute_buffer("source", "u_ref_angle", source_u_ref_angle.data());
-        result.add_attribute_buffer("source", "sk", source_sk.data());
-
-        result.add_buffer("sym_load", std::ssize(sym_load_id), std::ssize(sym_load_id), nullptr, nullptr);
-        result.add_attribute_buffer("sym_load", "id", sym_load_id.data());
-        result.add_attribute_buffer("sym_load", "node", sym_load_node.data());
-        result.add_attribute_buffer("sym_load", "status", sym_load_status.data());
-        result.add_attribute_buffer("sym_load", "type", sym_load_type.data());
-        result.add_attribute_buffer("sym_load", "p_specified", sym_load_p_specified.data());
-        result.add_attribute_buffer("sym_load", "q_specified", sym_load_q_specified.data());
-
-        result.add_buffer("asym_load", std::ssize(asym_load_id), std::ssize(asym_load_id), nullptr, nullptr);
-        result.add_attribute_buffer("asym_load", "id", asym_load_id.data());
-        result.add_attribute_buffer("asym_load", "node", asym_load_node.data());
-        result.add_attribute_buffer("asym_load", "status", asym_load_status.data());
-        result.add_attribute_buffer("asym_load", "type", asym_load_type.data());
-        result.add_attribute_buffer("asym_load", "p_specified", asym_load_p_specified.data());
-        result.add_attribute_buffer("asym_load", "q_specified", asym_load_q_specified.data());
-
-        result.add_buffer("shunt", std::ssize(shunt_id), std::ssize(shunt_id), nullptr, nullptr);
-        result.add_attribute_buffer("shunt", "id", shunt_id.data());
-        result.add_attribute_buffer("shunt", "node", shunt_node.data());
-        result.add_attribute_buffer("shunt", "status", shunt_status.data());
-        result.add_attribute_buffer("shunt", "g1", shunt_g1.data());
-        result.add_attribute_buffer("shunt", "b1", shunt_b1.data());
-        result.add_attribute_buffer("shunt", "g0", shunt_g0.data());
-        result.add_attribute_buffer("shunt", "b0", shunt_b0.data());
-
-        return result;
+auto const state_json = R"json({
```

Comment:
Shouldn't this be named `ref_state_json` or `complete_state_json`? The rest of the test would put it up against something else that has a prefix to its name, giving the impression that this is the base for others.